### DISCUSSION TITLE
What ffmpeg parameters are needed to stream the playlists?

### DISCUSSION THREAD

**Discussion by 2fst4u** [iptv-org/community#592] [Q&A]
I'm using an m3u aggregator, specifically threadfin which is a fork of xteve, to stream these links to my tv and ultimately to Plex.

But I can't get these streams to play in threadfin, I have other playlists from other places playing just fine but I get ffmpeg errors when using this list.

Any ideas?

**Comment by vednolacni** [comment]
> I'm using an m3u aggregator, specifically threadfin which is a fork of xteve, to stream these links to my tv and ultimately to Plex.
> 
> But I can't get these streams to play in threadfin, I have other playlists from other places playing just fine but I get ffmpeg errors when using this list.
> 
> Any ideas?

Did you figure it out?

**Comment by vager88** [comment]
Maybe you can try this..  -hide_banner -loglevel error -i [URL] -c:a libmp3lame -vcodec copy -f mpegts pipe:1
I'm currently using this in xTeve  for my connections. Seems to be working for me.
